src/optimization/iLP.jl

```diff
@@ -55,17 +56,17 @@ function solve(solver::ILP, problem::Problem)
         x = value.(first(neurons))
         matched, index = match_activation(nnet, x, δ)
         if matched
-            return interpret_result(solver, value(o), problem.input)
+            return interpret_result(solver, x, problem.input)
         end
         add_constraint!(model, nnet, neurons, δ, index)
     end
 end
 
-function interpret_result(solver::ILP, o, input)
-    if o >= maximum(input.radius)
-        return AdversarialResult(:holds, o)
+function interpret_result(solver::ILP, x, input)
+    if prod(abs.(x - input.center) .>= input.radius)
```

Comment:
This is false if any of the lhs < rhs. So we can do 
`all(abs.(x - input.center) .>= input.radius)`